Makefile

```diff
@@ -33,23 +39,46 @@ pre-pr: tidy generate lint test
 clean:
 	-@rm test/testdata/x.wasm
 
-sdk_srcs := $(wildcard ./basm/**/*.go)
-wasm_src_dir := $(shell realpath ./test/testdata)
-wasm_src := $(wasm_src_dir)/main.go
-wasm_out := $(wasm_src_dir)/x.wasm
+sdk_src := $(wildcard ./basm/**/*.go)
```

Comment:
This looks like it will miss `x/xbasm/**.go`

Reply:
thanks for catching. I updated to include x/**/*.go